.github/workflows/build.yml

```diff
@@ -57,8 +57,11 @@ jobs:
         run: echo LD_LIBRARY_PATH=${pythonLocation}/lib >> $GITHUB_ENV
 
       - if: inputs.msrv == 'MSRV'
-        name: Prepare minimal package versions (MSRV only)
-        run: nox -s set-minimal-package-versions
+        name: Prepare package versions for MSRV
+        run: |
+          sed -i '/pyo3-ffi-check/d' Cargo.toml  # we allow pyo3-ffi-check to use newer packages, as an internal tool
```

Comment:
I think the Noxfile already mentions `pyo3-ffi-check` by name, excluding it from linting.

So I see two alternatives to this "open-coded" approach:
* We move this change into the `set-minimal-package-versions` session to concentrate this in one place.
* We fully remove pyo3-ffi-check from the workspace, e.g. by adding an empty `[workspace]` to its `Cargo.toml`, and build it only when needed.

Personally, I would prefer the second option as the cleaner one.

Reply:
That's probably cleaner; I've force-pushed to reimplement as the second option.